stdlib/char.mc

```diff
@@ -1,78 +1,78 @@
 include "seq.mc"
 
-let eqchar = lam c1. lam c2. eqi (char2int c1) (char2int c2)
-let neqchar = lam c1. lam c2. neqi (char2int c1) (char2int c2)
-let ltchar = lam c1. lam c2. lti (char2int c1) (char2int c2)
-let gtchar = lam c1. lam c2. gti (char2int c1) (char2int c2)
-let leqchar = lam c1. lam c2. leqi (char2int c1) (char2int c2)
-let geqchar = lam c1. lam c2. geqi (char2int c1) (char2int c2)
-let show_char = lam c. concat "'" (concat [c] "'")
+let eqchar = lam c1. lam c2. eqi (charToInt c1) (charToInt c2)
+let neqchar = lam c1. lam c2. neqi (charToInt c1) (charToInt c2)
+let ltchar = lam c1. lam c2. lti (charToInt c1) (charToInt c2)
+let gtchar = lam c1. lam c2. gti (charToInt c1) (charToInt c2)
+let leqchar = lam c1. lam c2. leqi (charToInt c1) (charToInt c2)
+let geqchar = lam c1. lam c2. geqi (charToInt c1) (charToInt c2)
+let showChar = lam c. concat "'" (concat [c] "'")
```

Comment:
Should this perhaps be `charToString` instead? Feels more right to me since we have `intToString` instead of `showInt`.